core/cpuid.c

```diff
@@ -35,10 +35,10 @@
 #define CPUID_CACHE_SIZE 6
 
 typedef struct cpuid_cache_t {
-    uint32_t data[CPUID_CACHE_SIZE];  // Host cached features
-    cpuid_t  host_supported;          // Physical CPU supported features
-    cpuid_t  hax_supported;           // Hypervisor supported features
-    bool     initialized;
+    uint32_t     data[CPUID_CACHE_SIZE];  // Host cached features
+    hax_cpuid_t  host_supported;          // Physical CPU supported features
```

Comment:
Is cpuid_t  present in NetBSD headers and renaming is to resolve intersection issues?

Reply:
yes

Reply:
There is no option to rename this. `_t` is a reserved namespace for POSIX/implementation and third party code must not reuse it for internal purposes.

Reply:
There are also several other namespace conflicts in the past, such as `vaddr_t`. They were patched accordingly.

Reply:
Got it. That makes sense. Could you help to update the patch fe1524c to ensure the code line length not exceed column 80? After replacing string, some lines exceed column 80 and need to format. Then how about reordering the patch sequence as fe1524c first and 9417d38 last? Thanks.

Reply:
I will do it.